src/protocol.py

```diff
@@ -337,7 +337,7 @@ def assembleAddrMessage(peerList):
 
 
 def assembleVersionMessage(
-    remoteHost, remotePort, participatingStreams, server=False, nodeid=None
+    remoteHost, remotePort, participatingStreams, dandelion_enabled, server=False, nodeid=None,
```

Comment:
We can have `dandelion_enabled` default to `True`. Then we can put it at the end of arguments as well.